### ISSUE TITLE
Pretrained models not available

### ISSUE THREAD

**Issue by ashutosh1919** [lordmartian/deep_avsr#8]
Can you please provide pretrained models for the project

**Comment by LordMartian** [comment]
I will be releasing in 2 to 3 weeks.

**Comment by arunm95** [comment]
With regards to the iterations of curriculum learning, would you be able to provide a ballpark of the WER you achieved before moving to the next iteration? I'm also wondering if you could clarify how to intialize the AV network with the trained audio and visual networks?

**Comment by lordmartian** [comment]
I may not be able to provide the values of WER after each iteration of curriculum learning. Instead, I am providing the values of validation set (preval for pretrain, val for train) WER after pretraining and training for all the three models. Note that these values are computed using greedy search.

| Model | After Pretraining | After Training |
|:-----:|:-----------------:|:--------------:|
|   AO  |       0.253       |      0.191     |
|   VO  |       0.769       |      0.696     |
|   AV  |       0.245       |      0.168     |

Regarding how to initialize AV model, you should train the AO and VO models first. Then initialize the AV audio encoder with AO encoder, AV video encoder with VO encoder and AV decoder with VO decoder.

**Comment by lordmartian** [comment]
@ashutosh1919 
The release of pretrained weights will take some time. Sorry about that.

**Comment by lordmartian** [comment]
I have released the pretrained weights.

**Comment by javadpeyman** [comment]
where is it?

> I have released the pretrained weights.

where is it?

**Comment by lordmartian** [comment]
From README:

> Please send an email at smeetrs<AT>iitb.ac.in from your institutional email ID for the pre-trained weights of the AO, VO and AV models. Place the weights of each model in the corresponding /final/models directory.
